example_queries/group_by/count.cc

```diff
@@ -0,0 +1,15 @@
+// udf_type: Aggregation
```

Comment:
histogram.cc might be a more appropriate name for this file.

Reply:
done.